### DISCUSSION TITLE
Extract list of all stars

### DISCUSSION THREAD

**Discussion by LarryNon** [Stellarium/stellarium#3405] [Q&A] [answered]
I'm new to Stellarium.  I'd like to get an extract of all the stars' absolute position (J2000) and associated data (magnitude, color, etc.) based on a defined maximum magnitude.

The "Positions" page is close with the "Bright double stars" category, but the exports generates only 1300 stars. 

I see the star catalog is in .cat files (in  .\stars\default folder), but stored in a binary format.  Is there a way to parse these files to extract the data?  Or any other way that I missed?

Thanks.

**Comment by alex-w** [comment, marked as answer]
Why you don’t use original catalogs? In any way format of .cat file described in User Guide.

**Reply by LarryNon** [reply]
I ended-up using the data directly from the HIPPARCOS-2 catalogue.  It was simpler than decoding Stellarium's internal format.  

Ref: https://www.cosmos.esa.int/web/hipparcos/hipparcos-2
